cjs/interface/event.js

```diff
@@ -20,6 +20,8 @@ const GlobalEvent = typeof Event === 'function' ?
     constructor(type, eventInitDict = {}) {
       this.type = type;
       this.bubbles = !!eventInitDict.bubbles;
+      this._stopPropagationFlag = false;
+      this._stopImmediatePropagationFlag = false;
```

Comment:
`_stopImmediatePropagationFlag` will be needed, though EventTarget from https://github.com/ungap/event-target don't currently support that feature I thought it was still interesting to add it.